packages/eslint-plugin/test/__file_snapshots__/types/no-useless-files/index.d.ts.lint

```diff
@@ -1,5 +1,10 @@
-No errors
+types/no-useless-files/index.d.ts
+  1:1  error  Packages should contain value components, not just types  @definitelytyped/no-type-only-packages
+
+✖ 1 problem (1 error, 0 warnings)
 
 ==== types/no-useless-files/index.d.ts ====
 
     export default "I am useful";
```

Comment:
this looks like a value to me. Am I wrong?

Reply:
Yep, this is a bug. I didn't notice.

Reply:
Fixed.